plugins/in_winevtlog/in_winevtlog.c

```diff
@@ -70,6 +71,16 @@ static int in_winevtlog_init(struct flb_input_instance *in,
 
     /* Set up total reading size threshold */
     ctx->total_size_threshold = DEFAULT_THRESHOLD_SIZE;
+    tmp = flb_input_get_property("threshold_size", in);
+    if (tmp != NULL) {
+        threshold_size = flb_utils_size_to_bytes(tmp);
+        if (threshold_size > DEFAULT_THRESHOLD_SIZE) {
```

Comment:
This conditional should ensure that the value lies within a reasonable minimum and maximum size rather than just ensuring that it's larger than the default value. Please add two appropriately named constants to constraint the value between 1 KB and 1.8 MB (ideally so it the data fits within a reasonably sized chunk, but feel free to make the limit larger if need be)

Reply:
Got it. I changed the parameter name to read_limit_per_cycle.